.gitattributes

```diff
@@ -0,0 +1,2 @@
+clinvar-xml-parser/src/main/java/uk/ac/ebi/eva/clinvar/model/jaxb/*.java -diff
```

Comment:
We actually need to be able to see the full diff. Most of the times the changes between versions don't require any other code changes, but sometimes the structure of the XML changes and we need to know where.

Reply:
Done!

However, I kept `clinvar-xml-parser/src/test/resources/ClinvarExample.xml` excluded from the diff because the test records will not necessarily be the same, so comparing them won't make much sense.